lib/modules/@apostrophecms/piece-type/src/apos/components/ApostrophePiecesInsertModal.vue

```diff
@@ -5,7 +5,7 @@
       <p>New {{ moduleOptions.label }}</p>
     </template>
     <template slot="body">
-      <ApostropheSchemaEditor :fields="moduleOptions.schema" v-model="pieceInfo" />
+      <AposSchema :schema="moduleOptions.schema" v-model="pieceInfo" />
```

Comment:
Shall I remove the component `ApostropheSchemaEditor` from the code?

Reply:
If it's not being used anymore, yes.

Reply:
+1

Reply:
Done